tests/test_processing.py

```diff
@@ -558,54 +565,59 @@ def test_check_run_steps(
         "nodestats_run",
         "ordered_tracing_run",
         "splining_run",
+        "curvature_run",
         "log_msg1",
         "log_msg2",
     ),
     [
         pytest.param(
-            False,
-            False,
-            False,
-            False,
-            False,
-            False,
-            False,
+            False,  # Filters
+            False,  # Grains
+            False,  # Grainstats
+            False,  # Disordered tracing
+            False,  # Nodestats
+            False,  # Ordered tracing
+            False,  # Splining
+            False,  # Curvature
             "You have not included running the initial filter stage.",
             "Please check your configuration file.",
             id="All stages are disabled",
         ),
         pytest.param(
-            True,
-            False,
-            False,
-            False,
-            False,
-            False,
-            False,
+            True,  # Filters
+            False,  # Grains
+            False,  # Grainstats
+            False,  # Disordered tracing
+            False,  # Nodestats
+            False,  # Ordered tracing
+            False,  # Splining
+            False,  # Curvature
             "Detection of grains disabled, GrainStats will not be run.",
             "",
             id="Only filtering enabled",
         ),
         pytest.param(
-            True,
-            True,
-            False,
-            False,
-            False,
-            False,
-            False,
+            True,  # Filters
+            True,  # Grains
+            False,  # Grainstats
+            False,  # Disordered tracing
+            False,  # Nodestats
+            False,  # Ordered tracing
+            False,  # Splining
+            False,  # Curvature
             "Calculation of grainstats disabled, returning empty dataframe and empty height_profiles.",
             "",
             id="Filtering and Grain enabled",
         ),
         pytest.param(
-            True,
-            True,
-            True,
-            False,
-            False,
-            False,
-            False,
+            True,  # Filter
+            True,  # Grains
+            True,  # Grainstats
+            False,  # Disordered Tracing
+            False,  # Nodestats
+            False,  # Ordered tracing
+            False,  # Splining
+            False,  # Curvature
```

Comment:
Do you think we should have a test that checks if the `Curvature` option is `True` which would require all prior steps?